code/src/components/Header.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+const Header= () => {
+  return (
+    <h1>
+        New albums & singles
+    </h1>
```

Comment:
We would recommend putting the h1 directly in the app instead of making a component.